server/utils/__tests__/date.spec.js

```diff
@@ -65,28 +65,6 @@ describe('DateUtils', () => {
 });
 
 describe('getDateSelectionFrom', () => {
-  it('defaults to generating the previous 12 months from today', () => {
```

Comment:
this defaulting functionality is never used and eslint now picks up with only having the first param as a default